CesiumGltf/src/FeatureIdTextureView.cpp

```diff
@@ -35,6 +39,13 @@ FeatureIdTextureView::FeatureIdTextureView(
     return;
```

Comment:
This comment should be removed since we are no longer ignoring the texture's sampler :D

https://github.com/CesiumGS/cesium-native/blob/df7bb9fa914c60ed082b5e562040f7eecfd9a81c/CesiumGltf/src/FeatureIdTextureView.cpp#L35